src/connectors/laut.fm.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.fm-player';
+
+Connector.artistSelector = '.player-display__meta .player-display__meta--artist';
+
+Connector.trackSelector = '.player-display__meta .player-display__meta--title';
```

Comment:
I think we need to add a filter to strip off enclosing `"` e.g. `"Violent Sun"` -> `Violent Sun`

Something like this should work:

```
const filter = new MetadataFilter({ track: removeEnclosingQuotes });

Connector.applyFilter(filter);

function removeEnclosingQuotes(track) {
		track = track,substring(1);
    return track.substring(0, track.length - 1);
}
```

Although I am sure @alexesprit knows one built in that will be better :+1:

Reply:
Instead of using `String.substring` we can use `String.slice`:
```js
function removeEnclosingQuotes(track) {
    return track.slice(1, -1);
}
```

but this function will remove any characters.

Reply:
@alexesprit thanks! 

From looking it always pads with `"` so I think this is fine. Or would you rather go explicit and asset char at pos and remove only then?

Reply:
@inverse Thanks for catching that quotes issue. Added a change to fix that through @alexesprit's suggestion.